pkg/crc/image/image.go

```diff
@@ -0,0 +1,112 @@
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"runtime"
+	"strings"
+
+	crcpreset "github.com/code-ready/crc/pkg/crc/preset"
+	"github.com/containers/image/v5/copy"
+	"github.com/containers/image/v5/signature"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/containers/image/v5/types"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+var (
+	registryURI = "quay.io"
+	orgName     = "crcont"
+)
+
+type OCIImageHandler struct {
```

Comment:
Not sure `OCI` is really useful here, we hardcode `docker:` for fetching from the remote repository, and use `dir:` for local storage. Maybe this is still stored in an OCI compliant way, but this really is not the most important part of the implementation, especially as we don't support many layer formats.

Reply:
I think it is good to be explicit around what kind of image format we are storing to dir and right now `quay.io` pulled images are in oci image format so better to have it.